src/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("Avenir-Book.ttf");
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+.wrapper {
+  position: relative;
+  margin: 20px 0;
+}
+
+.decision {
+  outline: none;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  transition: box-shadow 0.2s;
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: inherit;
```

Comment:
when the user starts typing the text inside input should be bold (not a placeholder text)